packages/client/src/feed/Feed.ts

```diff
@@ -44,6 +50,7 @@ export class Feed {
   async fetch(
     request: FeedRequest,
     observerId?: string,
+    mediaTransformParams: MediaTransformParams = defaultMediaTransformParams,
```

Comment:
@cesarenaldi Are we okay with just allowing you to have one transformed image?

Reply:
in react-sdk PR we decided to introduce separate image sets for Publication and two sets for Profile (picture and cover) 
see: https://github.com/lens-protocol/lens-sdk/pull/430/files#diff-b89498756162aec8b024c715e6744fceac12f4529c23d825c4baafba133e2f45

I wanted to keep it simple with client sdk, but we could do sth similar here too

Reply:
I rewrite the whole thing, could you take a look again?